frontend/app/routes/protected/person-case/privacy-statement.tsx

```diff
@@ -70,6 +71,21 @@ export default function PrivacyStatement({ loaderData, actionData, params }: Rou
 
   return (
     <>
+      <div className="float-right">
```

Comment:
I think the more idiomatic way to render things far-right would be to use flexbox: `      <div className="flex justify-end">`